.reek

```diff
@@ -2,5 +2,9 @@ exclude_paths:
   - db/migrate
   - spec/**/*
 
+Attribute:
+  enabled: false
```

Comment:
I have an alternative solution that doesn't require disabling this. Will post after lunch.